unordered-containers.cabal

```
@@ -197,7 +197,7 @@ benchmark benchmarks
     bytestring,
     containers,
     gauge >= 0.2.5 && < 0.3,
-    deepseq >= 1.4,
+    deepseq >= 1.4.3,
```

Let's remove the now-redundant constraint

```suggestion
    deepseq,
```